public/planner/css/style.css

```diff
@@ -9,6 +9,21 @@ body.dz-drag-hover {
     opacity: .4;
 }
 
+.tooltip {
+    font-family: "Helvetica Neue", Helvetica;
+    display: block;
+    clear: both;
+    line-height: 1.5;
+    width: fit-content;
+    position: relative;
+    background: #555555;
+    color: #CCCCCC;
```

Comment:
The fonts size and color should match what's used in the UI in general.

Reply:
We tried to match the object count popup for the UI, is there an element in particular you would like the tooltip to match?
![image](https://user-images.githubusercontent.com/71036768/234979933-0bd59e62-c690-4057-9b26-5b47ca374025.png)

Reply:
Fair idea, but Object counter really isn't a default planner UI so I'd rather have the tooltips look like the menu items on top of the page. So a bit smaller font and white color. 
![Uploading Screenshot 2023-04-28 at 02.19.14.png…]()